course_content/notebooks/numpy_intro.ipynb

```diff
@@ -921,30 +922,45 @@
      "cell_type": "markdown",
      "metadata": {},
      "source": [
-      "Used without any further arguments, statistical functions simply reduce the whole array to a single value.  However, in practice we very often want to calculate over only *some* of the dimensions.  \n",
+      "Used without any further arguments, statistical functions simply reduce the whole array to a single value.  In practice, however, we very often want to calculate statistics over only *some* of the dimensions.\n",
+      "\n",
       "For example, given the above `daily_records` data, we might want :\n",
-      "  * (a) an average over all days, for each station *-- or --*\n",
-      "  * (b) an average over all stations, for each day."
+      "\n",
+      "* (a) an average over all days, for each station, or\n",
```

Comment:
Hm. We replaced these with number-lists above; this one seems to have slipped through. It's a shame markdown does not support letter-lists.